### DISCUSSION TITLE
Change name

### DISCUSSION THREAD

**Discussion by hatamiarash7** [renovatebot/renovate#16048] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

In some cases, we will use the `${{ github.actor }}` for Docker image names. This works well for other tools but not for Renovate. It's because of bot's name : **renovate[bot]**. For example:

![image](https://user-images.githubusercontent.com/16325641/173324686-a9980063-6688-4a3a-a5d0-cddbcf649129.png)

I think you should change the name of the bot to another thing, like : 

- renovate
- renovate-bot

It is essential that its name is in line with Docker rules.

> A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
I have converted this from issue to discussion. In future, please refrain from creating bug reports and instead create discussions first.

What you are requesting is impossible, because it's GitHub itself which assigns the name/actor value. Any app on GitHub has the text `[bot]` appended to it.

But, even if we could change the name, I don't think this is any good reason to do so.
